daphne_service_utils/src/lib.rs

```diff
@@ -0,0 +1,47 @@
+use std::str::FromStr;
+
+use serde::{Deserialize, Serialize};
+
+pub mod auth;
+pub mod metrics;
+pub mod test_route_types;
+
+/// Parameters used by the Leader to select a set of reports for aggregation.
+#[derive(Debug, Deserialize, Serialize)]
+pub struct DaphneServiceReportSelector {
+    /// Maximum number of aggregation jobs to process at once.
+    pub max_agg_jobs: u64,
+
+    /// Maximum number of reports to drain for each aggregation job.
+    pub max_reports: u64,
+}
+
+#[derive(Clone, Copy, Debug, Deserialize, PartialEq, Eq)]
+#[serde(rename_all = "snake_case")]
+pub enum Role {
```

Comment:
1. For consistency with `DapTaskConfig` et al
2. Daphne may, in the future, implement other protocols besides DAP.
```suggestion
pub enum DapRole {
```